src/testScripts/sdkV2-scripts.ts

```diff
@@ -0,0 +1,87 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import * as dotenv from 'dotenv';
+
+import { ethers } from 'ethers';
+import { CegaEvmSDKV2, EthereumAlchemyGasStation, types, GasStation } from '..';
+
+dotenv.config();
+
+const config = {
+  ethereum: {
+    RPC_URL: process.env.ETH_RPC_URL,
+    cegaEntryAddress: '0x0730AA138062D8Cc54510aa939b533ba7c30f26B' as types.EvmAddress,
+    usdcAddress: '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48' as types.EvmAddress,
+    gasStation: new EthereumAlchemyGasStation(process.env.ETH_ALCHEMY_API_KEY || ''),
+  },
+  arbitrum: {
+    RPC_URL: process.env.ARBITRUM_RPC_URL,
+    cegaEntryAddress: '0x10a5524f7c4e2fc62a1106e77cb8a53026bca252' as types.EvmAddress,
+    usdcAddress: '0xaf88d065e77c8cC2239327C5EDb3A432268e5831' as types.EvmAddress,
+    gasStation: new GasStation(),
+  },
+};
+
+const CURRENT_NETWORK = 'arbitrum';
+const { cegaEntryAddress, RPC_URL, gasStation } = config[CURRENT_NETWORK];
+
+const ADMIN_ACCOUNTS = {
+  programAdminPk: process.env.PROGRAM_ADMIN_PK || '',
+  operatorAdminPk: process.env.OPERATOR_ADMIN_PK || '',
+  traderAdminPk: process.env.TRADER_ADMIN_PK || '',
+  serviceAdminPk: process.env.SERVICE_ADMIN_PK || '',
+  userPk: process.env.RANDOM_USER_PK || '',
+};
+
+async function addDeposits() {
+  const provider = new ethers.providers.JsonRpcProvider(RPC_URL);
+  const userSigner = new ethers.Wallet(ADMIN_ACCOUNTS.userPk, provider);
+
+  const sdk = new CegaEvmSDKV2(cegaEntryAddress, gasStation, provider, userSigner);
+
+  // console.log('loaded sdk for add deposit');
```

Comment:
Do you mean to leave these in here?

Reply:
for the test script I've just left it pretty scrappy for now. because there was a bunch of random testing work that was done for this

Reply:
+1 to Vic, would be good to remove the logs and comments we aren't using

Reply:
removed a few comments/logs.
kept a few as, especially for transactions, we don't want to repeatedly run them each time, but we want to be able to run that script when needed.

if it starts to get messy for whoevers developing, we can just sweep clean it out, but feel we need a playground of these scripts to test that things work easily at least for the next 1-2 weeks